pandas/tests/reshape/concat/test_series.py

```diff
@@ -52,7 +46,7 @@ def test_concat_empty_and_non_empty_series_regression(self):
         result = pd.concat([s1, s2])
         tm.assert_series_equal(result, expected)
 
-    def test_concat_series_axis1(self, sort=sort):
+    def test_concat_series_axis1(self, sort=True):
         ts = tm.makeTimeSeries()
 
```

Comment:
I don't know about this, after moving sort to conftest.py, this test was giving me error, so i have changed default to True. Is this good, or do i have to move fixture to this file and revert this change?

Reply:
pls revert this, you should not be changing *any* code (literally just removing the fixture definition itself)

Reply:
ok, changed.